From: Takashi Iwai <tiwai@suse.de>
Subject: Add kABI placeholders for ASoC
Patch-mainline: Never, kABI padding
References: bsn#129 bsc#1179531
Modified-by-SEL: No

Add a few kABI padding fields to ASoC objects that are embedded in
other structs.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 include/sound/soc-component.h |    2 ++
 include/sound/soc.h           |    2 ++
 2 files changed, 4 insertions(+)

--- a/include/sound/soc-component.h
+++ b/include/sound/soc-component.h
@@ -221,6 +221,8 @@ struct snd_soc_component {
 	struct dentry *debugfs_root;
 	const char *debugfs_prefix;
 #endif
+
+	void *suse_kabi_padding;	/* XXX SLE-specific kABI placeholder */
 };
 
 #define for_each_component_dais(component, dai)\
--- a/include/sound/soc.h
+++ b/include/sound/soc.h
@@ -1125,6 +1125,8 @@ struct snd_soc_pcm_runtime {
 	struct snd_soc_dai_link *dai_link;
 	struct snd_pcm_ops ops;
 
+	void *suse_kabi_padding;	/* XXX SLE-specific kABI placeholder */
+
 	unsigned int params_select; /* currently selected param for dai link */
 
 	/* Dynamic PCM BE runtime data */
